papaya-web/src/main/java/pl/edu/agh/papaya/util/ConflictException.java

```diff
@@ -0,0 +1,16 @@
+package pl.edu.agh.papaya.util;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.ResponseStatus;
+
+@ResponseStatus(value = HttpStatus.CONFLICT)
```

Comment:
Czy to jest dobry kod błędu? Ja tego nie wiem, ale taki wylosowałam

Reply:
ja bym powiedział, że jest dobry :)